### PR TITLE
add priorities

### PR DESCRIPTION
**Pull request by tip3x**

### CODE REVIEW

File: charts/tensorleap/charts/engine/templates/engine-warmup-job.yaml [hunk 1]
```diff
@@ -27,3 +27,4 @@ data:
                 priorityClassName: "warmup-priority"
                 nodeSelector: warmup-nodeSelector
                 tolerations: warmup-tolerations
+                priorityClassName: medium-priority
```
Comment (yotamazriel) [inline charts/tensorleap/charts/engine/templates/engine-warmup-job.yaml:30]: This one needs to be lower than the low

**Review by yotamazriel — APPROVED** [review]
LGTM take a look on my comment

File: charts/tensorleap/charts/engine/templates/redis-deployment.yaml [hunk 2]
```diff
@@ -13,10 +13,10 @@ spec:
...
-        image: docker.io/library/redis:latest
-        ports:
-        - containerPort: 6379
+        - name: redis
+          resources:
+            requests:
+              memory: 1Gi
+          image: docker.io/library/redis:latest
+          ports:
+          - containerPort: 6379
```
Comment (asafyehezkel) [inline charts/tensorleap/charts/engine/templates/redis-deployment.yaml:22]: does the additional white space is purpose?

**Review by asafyehezkel — APPROVED** [review]
LGTM

**Review by yotamazriel — APPROVED** [review]
LGTM

**Pull request merged**